web/src/lib/cms/previews/RaffleBingoSwitchComponent.tsx

```diff
@@ -0,0 +1,16 @@
+import { RaffleBingoPaginator } from "@/components/tasks/RaffleBingoPaginator";
+import { Task } from "@/lib/types/types";
+import { rswitch } from "@/lib/utils/helpers";
+import { ReactElement } from "react";
+
+interface Props {
+  task: Task;
+}
+
+export const RaffleBingoSwitchComponent = (props: Props): ReactElement => {
+  return rswitch(props.task.filename, {
+    "raffle-license-step-1": <RaffleBingoPaginator task={props.task} CMS_ONLY_STEPINDEX={0} />,
+    "raffle-license-step-2": <RaffleBingoPaginator task={props.task} CMS_ONLY_STEPINDEX={1} />,
+    default: <RaffleBingoPaginator task={props.task} />,
+  });
+};
```

Comment:
[pebble] Does this really need to be its own component? The logic makes sense. I just don't see a reason we couldn't be handling this in the context of `RaffleBingoPreview`.